.markdownlint.json

```diff
@@ -7,7 +7,7 @@
   },
   "MD013": {
     "code_blocks": false,
-    "line_length": 120,
+    "line_length": 300,
```

Comment:
Please let's keep this down to 120. This is for readability when developing in an IDE as well as reviewing in PRs. It does make it a little annoying to maintain but it's a good practice to maintain.

Reply:
Okay...
Updated